data/drak/drak ships.txt

```diff
@@ -113,6 +113,9 @@ ship "Archon" "Archon (Cloaked)"
 	"never disabled"
 	add attributes
 		"cloak" .04
+		"cloaked communication" 1
+		"cloak phasing" 1
+		"cloaked scanning" 1
```

Comment:
```suggestion
```
Archons do not scan or possess the ability to scan, so I'm not sure it makes sense for them to have `cloaked scanning`.

Reply:
If nothing else, it indicates that they have detailed spacial awareness while cloaked; regardless of whether or not they are actually scanning in the mechanical gameplay sense. Given how aware the Archons appear to be, this seems appropriate. If anything, it seems more than the Archon's level of "scanning" transcends the bounds and mechanics that everyone else in the galaxy operates under. Like the difference between directly watching something vs merely watching a video that is just a lot of still images taken at a high rate of speed.

Also, I didn't want the Remnant to have a cloaking capability that the Drak didn't already have.

Reply:
Cloaked scanning would be useful if we defined Remnant ships as atrocities for the Nenia Archons (and remove them the moment when the Archons stop being mad at you). When the player would enter Nenia with a Remnant ship, they could get a special conversation ~~and be turned into space dust~~.